pipenv/cli.py

```diff
@@ -386,6 +386,16 @@ def parse_install_output(output):
     return names
 
 
+def get_python_from_requires():
+    python_version = project.parsed_pipfile.get('requires', {}).get('python_version')
+
+    if python_version:
+        for path in os.environ["PATH"].split(os.pathsep):
+            executable = os.path.join(path, "python%s" % python_version[:3])
```

Comment:
This also doesn't fail gracefully for poorly formatted/bad values in `python_version`. We should probably handle that rather than blowing up on `python_version[:3]` with an less helpful traceback.